tests/tests_plotting/test_misc.py

```diff
@@ -26,6 +39,6 @@ def test_plot_inner_solar_system_static(earth_perihelion):
 
 @pytest.mark.mpl_image_compare
 def test_plot_outer_solar_system_static(earth_perihelion):
-    plot_solar_system(outer=True, epoch=earth_perihelion)
+    plot_solar_system(outer=True, epoch=Time("1934-07-14 07:59", scale="tdb"))
```

Comment:
From `conftest.py`, `earth_perihelion` is `"2020-01-05 07:47:00"`. I get that, by using an epoch in the past, we avoid going outside the range of `epv00`.

However, I think that the correct solution here is to silence such warnings inside `plot_solar_system`. Yes, we know that getting the position of the Earth 100 years from now is unreliable, but for plotting we don't care.